actgen/evaluate_dqn.py

```diff
@@ -28,69 +28,22 @@ def __init__(self, test=True):
         if self.params['test'] or test:
             self.params['test'] = True
         self.setup()
-
+    
     def parse_args(self):
-        parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-        # yapf: disable
-        parser.add_argument('--env_name', type=str, default='CartPole-v0',
-                            choices=['CartPole-v0', 'Pendulum-v0', 'LunarLander-v2'],
-                            help='Which gym environment to use')
-        parser.add_argument('--agent', type=str, default='dqn',
-                            choices=['dqn', 'action_dqn'],
-                            help='Which agent to use')
-        parser.add_argument('--duplicate', '-d', type=int, default=5,
-                            help='Number of times to duplicate actions')
-        parser.add_argument('--random_actions', default=False, action='store_true',
-                            help='Make the duplicate actions all random actions')
-        parser.add_argument('--seed', '-s', type=int, default=0,
-                            help='Random seed')
-        parser.add_argument('--hyperparams', type=str, default='hyperparams/defaults.csv',
-                            help='Path to hyperparameters csv file')
-        parser.add_argument('--test', default=False, action='store_true',
-                            help='Enable test mode for quickly checking configuration works')
-        parser.add_argument('--results_dir', type=str, default='./results/',
-                            help='Path to the result directory to save model files')
-        parser.add_argument('--tag', type=str, default='default_exp',
-                            help='A tag for the current experiment, used as a subdirectory name for saving models')
-        parser.add_argument('--disable_gpu', default=False, action='store_true',
-                            help='enforce training on CPU')
-        args, unknown = parser.parse_known_args()
-        other_args = {
-            (utils.remove_prefix(key, '--'), val)
-            for (key, val) in zip(unknown[::2], unknown[1::2])
-        }
-        args.other_args = other_args
-        # yapf: enable
+        eval_parser = argparse.ArgumentParser(
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            parents=[self.parse_common_args()]
+        )
+        args = self.parse_unknown_args(eval_parser)
```

Comment:
Change function names. Here's an example of where the existing names are confusing.